src/figuresClasses.ts

```diff
@@ -2,10 +1,75 @@
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle' as const;
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('All sides must be positive numbers');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
```

Comment:
Similar to the previous comment, use `this.a`, `this.b`, and `this.c` to access the instance variables.